examples/remix/README.md

```diff
@@ -0,0 +1,186 @@
+# Use with Remix
+
+This example demonstrates how to use `jest-preview` with [`Remix`](https://remix.run/), bootstrapped using [`create-remix`](https://remix.run/docs/en/v1#getting-started) (without using any built-in templates/stacks).
+
+🚧 This example only demonstrates the simplest use of CSS in a Remix app (global stylesheet). Remix supports [many different ways of styling](https://remix.run/docs/en/v1/guides/styling). Example of more complex ways to style is coming.
+
+## Install Jest
+
+You have to manually install Jest in your Remix app.
```

Comment:
I think the main concentrate of this guide is how to integrate `Jest Preview` to `remix`. However, they need to configure Jest to work with Remix first. I suggest you create a repository/ blog post on your own to guide others to setup Jest with Remix standalone, then refer to the link here.

In this guideline, just keep it as concise as possible.